### PR TITLE
fix #33711 nudecelebs.pro

### LINKED ISSUE #33711: nudecelebs.pro
### Issue URL (Ads)
[NSFW][https://nudecelebs.pro/golaya-olga-buzova/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fnudecelebs.pro%2Fgolaya-olga-buzova%2F)
### Comment
Username: @Noisia12
> Custom userscripts:
> https://cdn.adguard.com/public/Userscripts/Beta/AdguardAssistant/4.2/assistant.user.js
> https://cdn.adguard.com/public/Userscripts/Beta/AdguardPopupBlocker/2.5/popupblocker.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.0.2578.6431
Browser: | Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:66.0) Gecko/20100101 Firefox/66.0
AdGuard driver: | WFP
Filters: | AdGuard Russian,<br>Filter unblocking search ads and self-promotion,<br>EasyList Dutch,<br>Liste FR,<br>ROList,<br>AdGuard Annoyances,<br>AdGuard French,<br>AdGuard Base,<br>ROLIST2,<br>road-block,<br>NoCoin Filter List,<br>Frellwit's Swedish Filter,<br>Dandelion Sprout's Nordic Filters,<br>AdGuard Tracking Protection,<br>AdGuard Social Media,<br>AdGuard Experimental,<br>AdGuard German,<br>AdGuard Japanese

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/33711

### CODE REVIEW

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -20,6 +20,8 @@ Ly9hbi55YW5kZXgucnUv
 !
 tut.by##.l-main > div[class^="b-top b-"] > div[class^="news-section-"]
 /images/bnn.gif$domain=nebesa.mobi
+||nudecelebs.pro/img/bg/*.gif
+nudecelebs.pro##.custom_block
```
Comment (BlazDT) [inline RussianFilter/sections/specific.txt]: Too generic name, better include parent elements.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply RussianFilter/sections/specific.txt]: I removed one obsolete rule and exchanged simple rule to ext-css.

**Review by BlazDT — APPROVED** [review]

**Comment by Alex-302** [comment]
missed ad
image

File: RussianFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -220,6 +220,7 @@ footballhd-news.com##div[id="dle-content"] > div.lastnewscat:has(> .adsbygoogle)
 !
 news.allcrimea.net#?##firstrubriks > div:has(> div > a[href="https://smi2.net/"])
 xn----8sbcm1bmcar0m.xn--p1ai#?#.td-post-content > div.custom-block:has(> noindex > table[border] > tbody > tr > td[align] > a[href*="moneynadom.ru"])
+nudecelebs.pro##article > div[class]:has(> div[style] > a[href*="bongacams3.com/"])
```
Comment (Alex-302) [inline RussianFilter/sections/css_extended.txt]: can't find it in html. Please show it on screenshot.

Comment (zloyden) [reply RussianFilter/sections/css_extended.txt]: <details><summary>Screenshot:</summary>

Снимок экрана 2019-08-06 в 10 23 06
</details><br/>

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**